test/bin/mdx-test/expect/env-and-prelude/test-case.md

```diff
@@ -0,0 +1,69 @@
+Different environments can have different preludes
+
+This is run with `--prelude default.ml --prelude a:a.ml --prelude b:b.ml`.
+Each of them defines a `y` variable, respectively set to `10`, `20` and `30`.
+
+```ocaml
+# let x = 1
+val x : int = 1
+```
+
+```ocaml env=a
+# let x = 2
+val x : int = 2
+```
+
+```ocaml env=b
+# let x = 3
+val x : int = 3
+```
+
+```ocaml
+# x + y
+- : int = 11
+# z
+- : int = 0
+```
+
+```ocaml env=a,version<4.08
```

Comment:
Maybe we can get rid of the two blocks with with version constraints and use an ellipse instead, what do you think?
```
# z
...
Error: Unbound value z
```